src/ctia/http/routes/crud.clj

```diff
@@ -197,6 +200,25 @@
                            search-filters
                            {:from s/Inst})
         aggregate-on-enumerable {:aggregate-on (apply s/enum (map name enumerable-fields))}
+        average-filters {:aggregate-on (describe (apply s/enum (map name (keys average-fields)))
+                                                 (apply str
+                                                        "The field to compute an average over. `from` and `to` are dates to narrow the "
+                                                        "entities used in this calculation.\n\n"
+                                                        (->> average-fields
+                                                             (map (fn [[aggregate-on {:keys [date-field]}]]
+                                                                    (str "When aggregate-on=" (name aggregate-on)
+                                                                         ", from/to filters on "
+                                                                         (case date-field
+                                                                           ;; :created is internal
+                                                                           :created " when the incident was first ingested in CTIA"
+                                                                           (name date-field))
+                                                                         ".")))
+                                                             (str/join " "))))
+                         :from (describe s/Inst "Start date of the average. See `aggregate-on` doc for which field is used to filter.")
+                         (s/optional-key :to) (describe s/Inst "End date of the average. See `aggregate-on` doc for which field is used to filter.")}
```

Comment:
we might need to factorize/normalize description between all aggregate metrics and search. Yours is the best, we need to reuse it :-).
Can you create an issue about that, please? We will tackle it later.

Reply:
https://github.com/advthreat/iroh/issues/7910